### PR TITLE
Chess board

### PR DESCRIPTION
**Pull request by Monchkrit**
Please merge whenever possible.

### CODE REVIEW

**Comment by Monchkrit** [comment]
Pieces have integer x and y spaces. That means the board has to support those coordinates, not standard chess spaces. I read that in Ken Mazaika's post on github as well. Another iteration is in order.

**Comment by Monchkrit** [comment]
Finally, I think I have the board set up correctly. This includes the images for all of the chess pieces.

File: app/controllers/boards_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class BoardsController < ApplicationController
```
Comment (ZoeBKramer) [inline app/controllers/boards_controller.rb:1]: Is there supposed to be a `create` method here? You have a create test as well as a create route but no create method inside the controller.

**Review by ZoeBKramer — COMMENTED** [review]
Was deleting the tests intentional? Just noticed a lot of stuff being removed.

File: app/controllers/boards_controller.rb (hunk 1 above)
Comment (Monchkrit) [reply app/controllers/boards_controller.rb:1]: Yes, https://online.bootcampspot.com/tdd/4

Comment (Monchkrit) [reply app/controllers/boards_controller.rb:1]: I am not doing the create. I'm just producing the board.

**Review by ZoeBKramer — APPROVED** [review]

**Pull request merged**